src/components/DocumentUtility/DocumentUtility.tsx

```diff
@@ -23,27 +22,17 @@ export const DocumentUtilityUnStyled = ({
   onPrint,
   className,
 }: DocumentUtilityProps) => {
-  const fileName = getData(document).name;
+  const fileName = getData(document)?.name ?? "Untitled";
   const qrcodeUrl = getData(document)?.links?.self?.href ?? "";
-  const logoUrl = getData(document)?.logo;
-  const [imageSettings, setImageSettings] = useState<ImageSettings>();
+  const imageSettings: ImageSettings = {
+    src: `/static/images/logo-qrcode.png`,
+    x: undefined,
+    y: undefined,
```

Comment:
how come need undef here? Cannot just dont describe it?

Reply:
actually its optional, let me omit them